scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -687,3 +687,39 @@ TDFN-6-1EP_2.5x2.5mm_P0.65mm_EP1.3x2mm:
   suffix: ''
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
+
+TDFN-14-1EP_3x3mm_P0.4mm_EP1.78x2.35mm:
+  device_type: 'TDFN'
+  size_source: 'https://pdfserv.maximintegrated.com/land_patterns/90-0063.PDF'
+  body_size_x: 3 +/-0.1
+  body_size_y: 3 +/-0.1
+  overall_height: 0.7
+  lead_width: 0.2 +/-0.02
+  lead_len: 0.7 +/-0.02
+
+  EP_size_x: 1.78 +/-0.02
+  EP_size_y: 2.35 +/-0.02
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_num_paste_pads: [2, 2]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
```

Comment:
Paste coverage is on the higher end.
Does Maxim recommend 75% somewhere?

Reply:
Based on QFN its 50% - 80% (https://www.maximintegrated.com/en/design/technical-documents/app-notes/6/6382.html) 
I guess I could go 65%?

Reply:
If there's no particular reason for a special value, please remove the line entirely.
That way, the generator can take care of it (and might be updated if a better default than 65% comes up)